tapis-project/tapis-ui
# task/TUI-51 - redux applications api

## Overview: ##

Add ApplicationsApi to redux, with support for applications lists

## Related Github Issues: ##

* [TUI-51](https://github.com/tapis-project/tapis-ui/issues/51)

## Summary of Changes: ##

- Added useApps hook
- Add AppsListing component
- Added unit tests for FileListing component (was missing)
- Slight changes to Systems (list action generator now accepts params object)

## Testing Steps: ##
1. npm install
2. npm run test
3. npm run dev
4. Log in as testuser2 and try to see apps

## UI Photos:

## Notes: ##
